RadClass/DiffSpectra.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+
+
+class DiffSpectra:
+    '''
+    Estimates a minimum background spectrum for each
+    timestamp/spectrum and saves a difference spectrum
+    between the two.
+
+    Attributes:
+    stride: The number of timestamps prior to the current
+        spectrum for finding the minimum background
+    '''
+
+    def __init__(self, stride=10):
+        self.stride = stride
+
+    def run(self, data):
+        # index all windows of length self.diff_stride into a tmp array
+        windows = [data[i-self.stride:self.stride] for i in range(self.stride-1, data.shape[0])]
```

Comment:
Don't you want this?
```suggestion
        windows = [data[i-self.stride:i] for i in range(self.stride-1, data.shape[0])]
```

Reply:
Also, doesn't the first index resolve to -1 for the first iteration?  Is that desired?

Reply:
No that is not desired. I must've been confused with my indexing. It should now reflect the desired behavior.